main/src/main/scala/sbt/Cross.scala

```diff
@@ -125,30 +126,65 @@ object Cross {
       case Left(cmd) => (resolveAggregates(x), cmd)
     }
 
+    val projCrossVersions = aggs map { proj =>
+      proj -> crossVersions(x, proj)
+    }
     // if we support scalaVersion, projVersions should be cached somewhere since
     // running ++2.11.1 is at the root level is going to mess with the scalaVersion for the aggregated subproj
```

Comment:
The comment probably can be removed now since we recover the state using `State` instead of issuing `++` at the end.